app/controllers/UserDataController.scala

```diff
@@ -99,6 +99,27 @@ class UserDataController(frontsApi: FrontsApi, dynamo: Dynamo, val deps: BaseFac
       Ok
     })
   }
+
+  def putFeatureSwitch() = APIAuthAction { request =>
+    val maybeFeatureSwitch: Option[FeatureSwitch] = request.body.asJson.flatMap(
```

Comment:
Similar to using Preferences, once we have less churn on this part of the tool I think this would be something good to look at, and worth sticking on the backlog.